solidstart-tanstackquery-tailwind-modules/msw/mockRepoInfo.ts

```diff
@@ -0,0 +1,7 @@
+import { rest } from "msw";
```

Comment:
why is this data here? It's not part of this ticket
please remove file

Reply:
I shouldn't have done this.